joss/paper.md

```diff
@@ -33,22 +33,25 @@ date: 10 June 2023
 bibliography: paper.bib
 ---
 
-# Summary
+# Introduction
 
 *In silico* evolution instantiates the processes of heredity, variation, and differential reproductive success (the three "ingredients" for evolution by natural selection) within digital populations of computational agents.
 Consequently, these populations undergo evolution [@pennock2007models], and can be used as virtual model systems for studying evolutionary dynamics.
 This experimental paradigm --- used across biological modeling, artificial life, and evolutionary computation --- complements research done using *in vitro* and *in vivo* systems by enabling experiments that would be impossible in the lab or field [@dolsonDigitalEvolutionEcology2021].
-Although visibility into *in vivo* systems is growing ever more fine-grained [@konno2022deep], complete, exact observability remains a key benefit of computational methods.
-For example, it is possible to perfectly record all parent-child relationships across simulation history, yielding complete phylogenies (ancestry trees).
+Although phylogenetic data from *in vivo* systems is growing ever more fine-grained [@konno2022deep], complete, exact observability remains a key benefit of computational methods.
+Indeed, in digital populations it is possible to perfectly record all parent-child relationships across history, yielding complete phylogenies (ancestry trees).
 This information reveals when traits were gained or lost, and also facilitates inference of underlying evolutionary dynamics [@mooers1997inferring;@dolsonInterpretingTapeLife2020;@moreno2024ecology].
 
-The Phylotrack project provides libraries for tracking and analyzing phylogenies in *in silico* evolution.
+Such capability is of great value to phylogenetics research. While backward-time models such as coalescent-based approaches are powerful, there are many scenarios that they cannot support, necessitating the use of forward-time evolutionary models [@hallerSLiMForwardGenetic2019]. Thus, research agendas that require predicting the phylogenetic patterns that would result from a given evolutionary process would benefit greatly from a way to track phylogenies in forward-time *in silico* evolution models or experiments. Currently, such tracking is implemented on a case-by-case basis for each individual digital model system. As efficient phylogenetic tracking is non-trivial to implement, this approach represents a research bottleneck.
```

Comment:
```suggestion
Such capability is of great value to phylogenetics research.
While backward-time models such as coalescent-based approaches are powerful, there are many scenarios that they cannot support, necessitating the use of forward-time evolutionary models [@hallerSLiMForwardGenetic2019].
Thus, research agendas that require predicting the phylogenetic patterns that would result from a given evolutionary process would benefit greatly from a way to track phylogenies in forward-time *in silico* evolution simulations. 
Currently, such tracking is implemented case-by-case in individual digital model systems.
As efficient phylogenetic tracking is non-trivial to implement, this \textit{status quo} burdens research workload.
```

Reply:
What if we just drop "simulations" from line 47 to side-step the whole instantiation/model/simulation/experiment philosophy of science discussion.

Reply:
Otherwise looks good!